pkgs/applications/version-management/gitlab/update.py

```diff
@@ -201,6 +201,16 @@ def update_gitlab_workhorse():
     _call_nix_update('gitlab-workhorse', gitlab_workhorse_version)
 
 
+@cli.command('update-gitlab-elasticsearch-worker')
+def update_gitlab_elasticsearch_worker():
+    """Update gitlab-elasticsearch-worker"""
+    data = _get_data_json()
+    gitlab_elasticsearch_worker_version = data['passthru']['GITLAB_ELASTICSEARCH_INDEXER_VERSION']
+    _call_nix_update('gitlab-elasticsearch-worker', gitlab_elasticsearch_worker_version)
+
+    subprocess.check_output(['./update-vendorHash.sh'], cwd='gitlab-elasticsearch-worker')
```

Comment:
[nix-update](https://github.com/Mic92/nix-update) already takes care of updating the vendor hash according to their [README](https://github.com/Mic92/nix-update/blob/master/README.md#features).

Reply:
That is correct